app/models/food.rb

```diff
@@ -1,3 +1,5 @@
 class Food < ApplicationRecord
   belongs_to :user
+has_many :orders
```

Comment:
The food should belong to an order

Reply:
in fact, I think the order should belong to the food and the user. because we need the food_id and user_id to create the order.